app/uk/gov/hmrc/transitmovementsrouter/models/MessageTypes.scala

```diff
@@ -64,19 +66,29 @@ object MessageType {
   // *******************
 
   /** E_DEC_AMD (IE013) */
-  case object DeclarationAmendment extends DepartureRequestMessageType("IE013", "CC013C")
+  case object DeclarationAmendment extends DepartureRequestMessageType("IE013", "CC013C") {
+    override val officeNode: String = "CustomsOfficeOfDeparture"
```

Comment:
Put this on `DepartureRequestMessageType` directly, rather than here, you've duplicated this unnecessarily.

Reply:
Agree. Will move up - same for ArrivalRequestMessageType.